libs/full/resiliency_distributed/include/hpx/resiliency_distributed/async_replay_distributed.hpp

```diff
@@ -66,8 +66,7 @@ namespace hpx { namespace resiliency { namespace experimental {
                 // necessary
                 auto this_ = this->shared_from_this();
                 return f.then(hpx::launch::sync,
-                    [this_ = std::move(this_), ids, iteration](
-                        hpx::future<Result>&& f) {
+                    [this_, ids, iteration](hpx::future<Result>&& f) {
```

Comment:
What's the reason for copying `this_` here rather than moving it? Or just capturing it directly as `[this_ = this->shared_from_this(), ...]`?

Reply:
Good catch, that's a left-over from previous changes. I will roll this back.